src/middlewares/isPasswordExpired.js

```diff
@@ -0,0 +1,15 @@
+import checkPasswordExpired from "../utils/isPasswordExpired";
+
+const isPasswordExpired = async (req, res, next) => {
+  const passwordUpdated = new Date(req.user.lastTimePasswordUpdated);
+  const expiredPassword = checkPasswordExpired(passwordUpdated);
+  if (expiredPassword) {
+    return res.status(401).json({
+      statu: 401,
+      message: "Your password has expired, Please update your password first!.",
```

Comment:
this function should include some error handling in case the value is invalid or missing.